template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ConsoleApplication-CSharp/Program.cs

```diff
@@ -1,2 +1,21 @@
-﻿Console.WriteLine("Hello, World!");
+﻿#if (csharpFeature_TopLevelProgram)
 // See https://aka.ms/new-console-template for more information
+#endif
+#if (!csharpFeature_ImplicitUsings)
+using System;
+
+#endif
+#if (!csharpFeature_TopLevelProgram)
```

Comment:
I would prefer if we don't negate if statement when we have `#else`

Reply:
Aye, it'd be great to invert it

Reply:
changed